apps/server/src/modules/files/job/delete-files.console.ts

```diff
@@ -1,5 +1,7 @@
-import { LegacyLogger } from '@src/core/logger';
 import { Command, Console } from 'nestjs-console';
+
+import { LegacyLogger } from '@src/core/logger';
+
```

Comment:
Why do you do new lines here?

Reply:
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4303#discussion_r1305536299